packages/botonic-nlp/src/preprocess/engines/en/tokenizer-en.ts

```diff
@@ -1,10 +1,8 @@
-import * as TreebankWordTokenizer from 'natural/lib/natural/tokenizers/treebank_word_tokenizer'
-
 import { Tokenizer } from '../../types'
 
-export default class TokenizerEn implements Tokenizer {
+export class TokenizerEn implements Tokenizer {
   readonly locale = 'en'
-  private tokenizer = new TreebankWordTokenizer()
+  private tokenizer = new (require('@nlpjs/lang-en-min/src/tokenizer-en'))()
```

Comment:
why do you do the require here?

Reply:
It could be done before but I didn't know which name to use for it so I decided to just use it directly. 
Maybe I could have used something like `NlpjsTokenizerEn` (`Tokenizer` and `TokenizerEn` were already in use).

Do you think refactor is needed? @dpinol 

Maybe to something like:
`const NlpjsTokenizerEn = require('@nlpjs/lang-en-min/src/tokenizer-en')`
`private tokenizer = new NlpjsTokenizerEn()`

Reply:
ok thanks, leave. Let's discuss it when merging with botonic/contentful Tokenizers